docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -87,19 +90,26 @@ The following example shows how to create a gallery-like flexbox layout using th
     content.JustifyContent = FlexContainer.Justification.JustifyCenter;
     /// Align items vertically center
     content.AlignItems = FlexContainer.Alignment.AlignCenter;
-    /// Occupy 90 percent of available space on the cross axis
-    content.Flex = 0.9f;
     /// Add the content area to the main container
     flexContainer.Add(content);
     ```
 
-7.  Add items to the toolbar.
+7.  Use `Flex` property of the `FlexContainer` class to specify the area to be occupied within the main container. Since the main container displays items vertically its height is shared between the toolbar and the content. Setting `toolbar.Flex` to 0.1 and `content.Flex` to 0.9 means that `toolbar` will occupy 10% of vertical space and `content` will occupy 90% of vertical space of the main container. The width of the items will be eqaul to the width of the main container:
```

Comment:
@pkalota slight correction in spelling and adding a , 
```suggestion
7.  Use `Flex` property of the `FlexContainer` class to specify the area to be occupied within the main container. Since the main container displays items vertically, its height is shared between the toolbar and the content. Setting `toolbar.Flex` to 0.1 and `content.Flex` to 0.9 means that `toolbar` will occupy 10% of vertical space and `content` will occupy 90% of vertical space of the main container. The width of the items will be equal to the width of the main container: